victor/src/main/groovy/com/trello/victor/VictorPluginExtension.groovy

```diff
@@ -42,4 +42,17 @@ class VictorPluginExtension {
      */
     List<String> excludeDensities = []
 
+    /**
+     * [Experimental]
+     *
+     * If set to true, Victor will generate Android vector drawables
+     * If not, it will generate PNGs in all specified densities
```

Comment:
I think we can merge this now, as long as we specify that it only supports a subset of SVGs (e.g. only `px` values).